source/Ship.cpp

```diff
@@ -2689,7 +2689,7 @@ double Ship::TransferFuel(double amount, Ship *to)
 void Ship::WasCaptured(const shared_ptr<Ship> &capturer)
 {
 	// Repair up to the point where this ship is just barely not disabled.
```

Comment:
This isn't extremely important, but I would change the comment here to something else.
```suggestion
	// Repair the ship so that it has a bit more than the minimum required hull, so that it isn't disabled immediatly afterwards.

Reply:
Also, "just barely not" is a bit wordy IMO.

Reply:
Well I feel like ships are still just barely not disabled even after this change. They're just not on a knifes edge between disabled and not. \:)